ui/src/containers/Search/useDebounce.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+import { useMst } from '../../store/root';
+
+export const UseDebounce = (value: string, delay: number) => {
```

Comment:
`UseDebounce` is a function and not a Functional Component right? Can we refractor this to `useDebounce`?